ui/controls_test.go

```diff
@@ -0,0 +1,38 @@
+package ui
+
+import (
+	"bytes"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/x/exp/teatest"
+	"testing"
+	"time"
+)
+
+func Test_ControlsViewRender(t *testing.T) {
+	expected := "(q)uit | (d)elete | (g)enerate | (t)xn | (h)ide"
+	// Create the Model
+	m := MakeControlViewModel()
+
+	tm := teatest.NewTestModel(
+		t, m,
+		teatest.WithInitialTermSize(80, 40),
```

Comment:
Would it be worth running these kinds of UI tests with different terminal sizes?

Reply:
Definitely not worth it. The test suite is too experimental to provide good results.